algoliasearch-apache-uber/pom.xml

```diff
@@ -48,9 +49,14 @@
                         <Bundle-SymbolicName>com.algolia.algoliasearch-apache-uber</Bundle-SymbolicName>
                         <Bundle-Name>algoliasearch-apache-uber</Bundle-Name>
                         <Bundle-Version>${project.version}</Bundle-Version>
-                        <Export-Package>com.algolia.search.*,${relocated.org.apache}.http.impl.nio.client</Export-Package>
+                        <Export-Package>
+                            com.algolia.search.*,
+                            ${relocated.org.apache}.http.impl.nio.client,
+                            ${relocated.com.fasterxml.jackson}.annotation,
+                            ${relocated.com.fasterxml.jackson}.databind
+                        </Export-Package>
                         <Import-Package>
-                            javax.net.ssl,javax.naming,!org.apache.avalon.framework.logger,!org.apache.log,*
+                            javax.net.ssl,javax.naming,!org.apache.avalon.framework.logger,!org.apache.log,!javax.servlet,*
```

Comment:
question: why are we dropping `!javax.servlet`?

Reply:
This first came up while doing my tests was Apache Karaf which does no provide `javax.servlet` (anymore?).
The bundle was not able to start because `javax.servlet` is a dependency and it is missing! 
However, the client does not actually need it to work! excluding it solved the issue.